### PR TITLE
Promote globe tools API to public

### PR DESCRIPTION
**Pull request by markschlosseratbentley**
Promote globe view tools to public.

### CODE REVIEW

**Comment by pmconne** [comment]
I added thematic to #1096 before you submitted this.

File: core/frontend/src/tools/ViewTool.ts [hunk 1]
```diff
@@ -3298,8 +3298,10 @@ export class ViewGlobeLocationTool extends ViewTool {
   public static get minArgs() { return 1; }
   public static get maxArgs() { return undefined; }
 
-  // arguments: latitude longitude | string
-  // the latitude and longitude arguments are specified in degrees
+  /** This runs the tool based on the provided location arguments.
+   * arguments: latitude longitude | string
+   * the latitude and longitude arguments are specified in degrees
+   **/
```
Comment (pmconne) [inline core/frontend/src/tools/ViewTool.ts:3304]: Elaborate on "location" and/or provide examples ("Syndney Opera House", "1600 Pennsylvania Drive, Washington DC", etc).

File: docs/changehistory/NextVersion.md [hunk 2]
```diff
@@ -91,6 +91,31 @@ The following APIs have been promoted to `public`. Public APIs are guaranteed to
 
 All hyper-modeling APIs are now public. [This interactive sample](https://www.itwinjs.org/sample-showcase/?group=Viewer&sample=hypermodeling-sample&imodel=House+Sample) demonstrates how to use hyper-modeling features.
 
+### [@bentley/imodeljs-frontend](https://www.itwinjs.org/reference/imodeljs-frontend/)
+
+* [ViewGlobeSatelliteTool]($frontend) for viewing the current camera location from approximately a satellite's height above the background map.
```
Comment (pmconne) [inline docs/changehistory/NextVersion.md:96]: Collapse these into one bullet point describing them all as tools for viewing the iModel in a global context.

**Review by pmconne — CHANGES_REQUESTED** [review]
Once the sample showcase integrates 2.14.0 and [my sample](https://github.com/imodeljs/frontend-sample-showcase/pull/161) completes, we should link it from your NextVersion.md description of the globe tools.

**Comment by markschlosseratbentley** [comment]
> I added thematic to #1096 before you submitted this.
> I don't think it's necessary to enumerate every type.

Thanks - I'll remove this from this PR.

File: docs/changehistory/NextVersion.md (hunk 2 above)
Comment (markschlosseratbentley) [reply docs/changehistory/NextVersion.md]: done

**Comment by markschlosseratbentley** [comment]
> > I added thematic to #1096 before you submitted this.
> > I don't think it's necessary to enumerate every type.
> 
> Thanks - I'll remove this from this PR.

@pmconne removed.

**Comment by pmconne** [comment]
> Thanks - I'll remove this from this PR.

@kabentley seems to want people to add to NextVersion.md on #1096. I'm not sure if that's better or worse than having each PR author resolve conflicts in that file themselves - it will mean his PR can't build/complete until everybody else's PRs complete. Maybe move your NextVersion.md changes over to there?

**Comment by markschlosseratbentley** [comment]
> > Thanks - I'll remove this from this PR.
> 
> @kabentley seems to want people to add to NextVersion.md on #1096. I'm not sure if that's better or worse than having each PR author resolve conflicts in that file themselves - it will mean his PR can't build/complete until everybody else's PRs complete. Maybe move your NextVersion.md changes over to there?

@pmconne All right.

File: core/frontend/src/tools/ViewTool.ts (hunk 1 above)
Comment (markschlosseratbentley) [reply core/frontend/src/tools/ViewTool.ts:3305]: good idea - thanks

Comment (markschlosseratbentley) [reply core/frontend/src/tools/ViewTool.ts:3305]: done

**Comment by markschlosseratbentley** [comment]
> > > Thanks - I'll remove this from this PR.
> > 
> > 
> > @kabentley seems to want people to add to NextVersion.md on #1096. I'm not sure if that's better or worse than having each PR author resolve conflicts in that file themselves - it will mean his PR can't build/complete until everybody else's PRs complete. Maybe move your NextVersion.md changes over to there?
> 
> @pmconne All right.

@pmconne I removed NextVersion.md entry and will put it on Keith's PR.

File: core/frontend/src/tools/ViewTool.ts [hunk 3]
```diff
@@ -3169,7 +3169,7 @@ export class FitViewTool extends ViewTool {
 }
 
 /** A tool that views a location on the background map from a satellite's perspective; the viewed location is derived from the position of the current camera's eye above the background map. Operates on the selected view.
- * @beta
+ * @public
```
Comment (pmconne) [inline core/frontend/src/tools/ViewTool.ts:3172]: The overridden methods like `onDataButtonDown` and `onPostInstall` will appear in docs without documentation. They're not very interesting/useful vs the documented base class methods, so perhaps they should be marked as internal. @kabentley disagree?

File: docs/changehistory/NextVersion.md (hunk 2 above)
Comment (markschlosseratbentley) [reply docs/changehistory/NextVersion.md]: This has been pushed to https://github.com/imodeljs/imodeljs/pull/1096

File: core/frontend/src/tools/ViewTool.ts (hunk 3 above)
Comment (markschlosseratbentley) [reply core/frontend/src/tools/ViewTool.ts:3172]: Any thoughts on this?

Comment (pmconne) [reply core/frontend/src/tools/ViewTool.ts:3172]: Please mark them `@internal`. They are part of the implementation, not of the API.

Comment (markschlosseratbentley) [reply core/frontend/src/tools/ViewTool.ts:3172]: For all view tools?  I'll limit it to my view globe tools for this PR - but lots of other ones have them untagged.

Comment (pmconne) [reply core/frontend/src/tools/ViewTool.ts:3172]: Yes - the ones affected by your PR.

Comment (markschlosseratbentley) [reply core/frontend/src/tools/ViewTool.ts:3172]: pushed this.

**Review by pmconne — APPROVED** [review]

File: core/frontend/src/tools/ViewTool.ts (hunk 3 above)
Comment (markschlosseratbentley) [reply core/frontend/src/tools/ViewTool.ts:3172]: resolve this conversation if looks good?

**Pull request merged**